conf/production.ini

```diff
@@ -52,12 +48,6 @@ level = INFO
 handlers =
 qualname = alembic
 
-[logger_sentry]
```

Comment:
Does getting rid of these affect our ability to disable logging/reporting of specific error messages to Sentry? (Essentially ignore certain errors.) I realize we don't do this in lms but we do use logging filters to do this in H so I was curious.

Reply:
Where do we do that in h? I haven't seen that. Anyway, h uses the legacy `raven` SDK, I haven't seen anything in the new `sentry-sdk` docs about logging configuration. I think if you wanted to ignore certain errors, you'd just do that in the Sentry UI? Not sure if it provides any way for your Python code to plug in to `sentry-sdk` and prevent certain exceptions from being sent to Sentry at all.

Reply:
h/config.py:120
```
    # Add ES logging filter to filter out ReadTimeout warnings
    es_logger = logging.getLogger('elasticsearch')
    es_logger.addFilter(ExceptionFilter((("ReadTimeoutError", "WARNING"),)))
```
Looks like this in the es logger so this prob would still work in that case. Assuming sentry
still only reports things that are logged? I guess that's the key here.